openvpn/pia/update-port.sh

```diff
@@ -27,14 +27,12 @@ pf_host=$(ip route | grep tun | grep -v src | head -1 | awk '{ print $3 }')
 ###### Nextgen PIA port forwarding      ##################
 
 get_auth_token () {
-            tok=$(curl --insecure --silent --show-error --request POST --max-time $curl_max_time \
-                 --header "Content-Type: application/json" \
-                 --data "{\"username\":\"$user\",\"password\":\"$pass\"}" \
-                "https://www.privateinternetaccess.com/api/client/v2/token" | jq -r '.token')
-            [ $? -ne 0 ] && echo "Failed to acquire new auth token" && exit 1
-            #echo "$tok"
-    }
-
+  tok=$(curl --silent --show-error --request POST --max-time $curl_max_time \
+      --user "$user:$pass" \
+      "https://www.privateinternetaccess.com/gtoken/generateToken" | jq -r '.token')
+  [ $? -ne 0 ] && echo "Failed to acquire new auth token" && exit 1
+  #echo "$tok"
```

Comment:
This is technically commented-out code, I'm happy to remove it, but since it was originally there, I thought I should leave it.